src/tiler/sql/user_reservable.sql

```diff
@@ -10,24 +10,29 @@
       block.geom,
       block.id,
       turf.group_id,
-      grp.slug AS group_slug,
+      core_group.slug AS group_slug,
       CASE
         WHEN (block.is_available AND reservation.id IS NULL
-              AND (turf.id IS NULL OR trustedmapper.id IS NOT NULL)) THEN 'available'
+              AND (turf.id IS NULL
+                    OR trustedmapper.id IS NOT NULL
+                    OR core_group.admin_id = core_user.id)) THEN 'available'
         ELSE 'unavailable'
       END AS survey_type,
       CASE
         WHEN reservation.id IS NOT NULL THEN 'reserved'
-        WHEN grp.id IS NOT NULL AND NOT grp.allows_individual_mappers THEN 'unavailable'
+        WHEN (core_group.id IS NOT NULL
+                AND NOT core_group.allows_individual_mappers) THEN 'unavailable'
         WHEN NOT block.is_available THEN 'unavailable'
-        WHEN turf.id IS NOT NULL AND trustedmapper.id IS NULL THEN 'group_territory'
+        WHEN (turf.id IS NOT NULL
+                AND NOT (trustedmapper.id IS NOT NULL
+                            OR core_group.admin_id = core_user.id)) THEN 'group_territory'
         ELSE 'none'
       END AS restriction
       FROM survey_blockface AS block
+      INNER JOIN core_user ON core_user.id = <%= user_id %>
       LEFT OUTER JOIN survey_territory AS turf
         ON block.id = turf.blockface_id
-      LEFT JOIN core_group AS grp
-        ON grp.id = turf.group_id
+      LEFT JOIN core_group ON core_group.id = turf.group_id
       LEFT OUTER JOIN survey_blockfacereservation AS reservation
         ON (block.id = reservation.blockface_id
             AND reservation.canceled_at IS NULL
```

Comment:
We should drop the join to `core_user` and replace references `core_user.id` with `<%= user_id %>`

We need to update the [cache buster](https://github.com/azavea/nyc-trees/blob/develop/src/nyc_trees/apps/survey/layer_context.py#L23-L25) for this query to include `Group` now.